templates/mails/mailing-welcome-subject.txt

```diff
@@ -1,2 +1 @@
-{% load i18n %}
-{% trans "Bienvenue sur la plateforme BIMData.io" %}
\ No newline at end of file
+{% load i18n %}{% trans "Bienvenue sur la plateforme BIMData.io" %}
```

Comment:
This works until we need an if/else or more complex template.
I suggest stripping the output of the template to remove `\n` like I did here https://github.com/bimdata/sso-invitation-email-sender/blob/master/src/app.py#L84